src/activities/Elsa.Activities.Rpa.Web/Activities/ClickElement.cs

```diff
@@ -0,0 +1,47 @@
+using Elsa.ActivityResults;
+using Elsa.Attributes;
+using Elsa.Design;
+using Elsa.Expressions;
+using Elsa.Services.Models;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Support.Extensions;
+
+// ReSharper disable once CheckNamespace
+namespace Elsa.Activities.Rpa.Web
+{
+    [Action(Category = "Rpa.Web", Description = "Clicks an element in the GUI")]
+    public class ClickElement : WebActivityWithSelector
+    {
+        public ClickElement(IServiceProvider sp) : base(sp)
+        {
+        }
+
+        
+        [ActivityInput(Hint = "Indicates whether not to perform an interactive click but just emulates a click via javascript call")]
+        public bool? UseJavascriptClick { get; set; }
+        protected override async ValueTask<IActivityExecutionResult> OnExecuteAsync(ActivityExecutionContext context)
+        {
+            try
+            {
+                var driver = _factory.GetDriver(GetDriverId(context));
+                if (UseJavascriptClick ?? false)
+                    driver.ExecuteJavaScript("arguments[0].click()", await GetElement(driver));
+                else
+                    (await GetElement(driver)).Click();
+                return Done();
+            }
+            catch (Exception e)
+            {
+                if (GetDriverId(context) != default)
+                {
+                    _factory.CloseBrowserAsync(GetDriverId(context)).Wait();
+                }
+                return Fault(e);
```

Comment:
I'm noticing this try/catch pattern in every activity, so I was wondering, since you already have a base class, if you might be able to provide a protected base method that does the try/catch while the child class provides the logic to execute?

e.g.

```csharp
public class WebActivity
{
   protected async Task<IActivityExecutionResult> ExecuteDriver(ActivityExecutionContext context, Action<IWebDriver> action)
   {
      var driverId = GetDriverId(context)
      try
            {
                var driver = _factory.GetDriver(driverId);
                action(driver);
               return Done();
            }
            catch (Exception e)
            {
                if (driverId != default)
                {
                    await _factory.CloseBrowserAsync(driverId);
                }
                return Fault(e);
           }
   }
}
```

Then use it like this:

```csharp
protected override async ValueTask<IActivityExecutionResult> OnExecuteAsync(ActivityExecutionContext context)
{
   return await ExecuteDriver(context, driver => 
   {
      if (UseJavascriptClick ?? false)
          driver.ExecuteJavaScript("arguments[0].click()", await GetElement(driver));
      else
          (await GetElement(driver)).Click();
   });
}
```